internal/manifests/manifests_test.go

```diff
@@ -949,6 +994,21 @@ spec:
 			Expect(listedManifests[0].FileName).To(Equal("user-generated-manifest.yaml"))
 		})
 	})
+
+	Context("GetUserManifestSuffixes", func() {
+		It("Should be able to list only user manifests if user and non user manifests are present", func() {
+			clusterId := registerCluster().ID
+			s3Metadata := []string{
+				filepath.Join(clusterId.String(), constants.ManifestMetadataFolder, "manifests", "first.yaml", constants.ManifestSourceUserSupplied),
+				filepath.Join(clusterId.String(), constants.ManifestMetadataFolder, "openshift", "second.yaml", constants.ManifestSourceUserSupplied),
+				filepath.Join(clusterId.String(), constants.ManifestMetadataFolder, "manifests", "third.yaml", "other-metadata"),
```

Comment:
I would not label this as "other-metadata", I would consider labeling "other-manifest-source" to make it clear that this is not just some arbitrary metadata field.